src/angular/sourceMappingVisitor.ts

```diff
@@ -104,6 +104,7 @@ export class SourceMappingVisitor extends RuleWalker {
 
   createFailure(s: number, l: number, message: string, fix?: Fix): RuleFailure {
     const { start, length } = this.getMappedInterval(s, l);
+    // tslint:disable-next-line:deprecation
     return super.createFailure(start, length, message, fix);
```

Comment:
`createFailure` is deprecated?

Reply:
Yep, it has been deprecated for over a year. PR https://github.com/palantir/tslint/pull/1952